pkgs/development/python-modules/pyqtwebengine/6.x.nix

```
@@ -0,0 +1,86 @@
+{ lib
+, pythonPackages
+, pkg-config
+, cmake
+, qtbase
+, qtwayland
+, qtwebengine
+, wrapQtAppsHook
+}:
+
+# based on pkgs/development/python-modules/pyqt/6.x.nix
+
+let
+  inherit (pythonPackages) buildPythonPackage python isPy27 pyqt6 enum34 sip-pyqt6 pyqt6-builder qmake2cmake;
+in buildPythonPackage rec {
```

```suggestion
with pythonPackages;

buildPythonPackage rec {
```

now we don't need to maintain that list.